libs/deposits/src/lib/use-submit-deposit.ts

```diff
@@ -57,7 +54,7 @@ export const useSubmitDeposit = (
   }, confirmations);
 
   useSubscription<DepositEvent, DepositEventVariables>(DEPOSIT_EVENT_SUB, {
-    variables: { partyId: partyId || '' },
+    variables: { partyId: partyId ? partyId.substring(2, partyId.length) : '' },
```

Comment:
(chore, non-blocking): I think this would benefit from being reusable, and should include a check that the string begins with '0x'.

Reply:
ahoy to that!
`const remove0x = s => s.replace(/^0x/, '')`